pkl-core/src/main/java/org/pkl/core/project/Project.java

```diff
@@ -325,7 +278,7 @@ private Project(
     return pkg;
   }
 
-  public EvaluatorSettings getSettings() {
+  public PklEvaluatorSettings getSettings() {
     return evaluatorSettings;
   }
```

Comment:
Per my comment below to have a deprecation cycle, we can do this:

```suggestion
  /** Use {@link org.pkl.core.project.Project#getEvaluatorSettings()} instead. */
  @Deprecated(forRemoval = true)
  public EvaluatorSettings getSettings() {
    return new EvaluatorSettings(evaluatorSettings);
  }

  public PklEvaluatorSettings getEvaluatorSettings() {
    return evaluatorSettings;
  }
```

This works out nicely because `getSettings()` should probably be renamed anyway.